go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/miekg/dns
 go 1.14
 
 require (
-	golang.org/x/net v0.0.0-20210726213435-c6fcb2dbf985
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
-	golang.org/x/tools v0.1.6-0.20210726203631-07bc1bf47fb2
+	golang.org/x/net v0.2.0
```

Comment:
probably not needed, but adding new RRs something results in a cycle, although pusing these to an actual release might not be bad

Reply:
Running `go generate` in a freshly clone of the repo yields this error:
```
2022/11/20 12:00:47 internal error: package "context" without types was imported from "github.com/miekg/dns"
exit status 1
duplicate.go:3: running "go": exit status 1
```
Bumping the dependencies does fix it on my system, for whatever reason. I have not figured out why that is.

Reply:
saw that also when adding new RRs, some kind of cycle between what you have locally and what's available online, never paid enough attention to it, to figure out what it is